Sources/Configuration/SDK.xcconfig

```diff
@@ -17,7 +17,6 @@ CLANG_ENABLE_OBJC_ARC = YES
 CLANG_ENABLE_MODULES = YES
 //OTHER_CFLAGS = -Wall -Wextra -Wpedantic -Wno-gnu-conditional-omitted-operand
 
-ONLY_ACTIVE_ARCH[config=Debug] = YES
```

Comment:
This did nothing so I just removed it as part of cleaning things up
![image](https://github.com/user-attachments/assets/6e87c163-4bc1-4c71-bc4d-968f88dc8ef5)